gateway-spi/src/main/java/org/apache/knox/gateway/services/security/token/impl/JWTToken.java

```diff
@@ -80,12 +82,17 @@ public JWTToken(String alg, String[] claimsArray, List<String> audiences, boolea
     if(claimsArray[3] != null) {
       builder = builder.expirationTime(new Date(Long.parseLong(claimsArray[3])));
     }
+    if(claimsArray[4] != null) {
```

Comment:
Are there any callers of this which may not have a claims array of this length? In other words, ought this check the length of the claimsArray before checking whether the value is null? I think it only gets used by Knox issuers.

Reply:
It does but that is a good test before we stumble into ArrayIndexOutOfBounds exception

Reply:
Here I just log and move on, since we are dealing with arrays of fix size initialized earlier in a different class there is no easy way to recover.